demo/utils/EditInCodeSandboxButton.vue

```diff
@@ -47,8 +47,20 @@ export default defineComponent({
     size: String
   },
   setup (props) {
+    const parameters = ref('')
+
+    watch(
+      () => props.code,
+      (value) => {
+        parameters.value = getCodeSandboxParams(value)
+      },
+      {
+        immediate: true
+      }
+    )
```

Comment:
We should use computed, not a watch like this

Reply:
You're right, I forget it.😅